practices/api_first_development

```diff
@@ -0,0 +1,94 @@
+---
+slug: api_first_development
```

Comment:
I have updated the contributing guide with more precise information about the format:

> slug - A friendly ID in [kebab-case](https://betterprogramming.pub/string-case-styles-camel-pascal-snake-and-kebab-case-981407998841#a084) format of the article visible in the URL of the Knowledge base (e.g.: for https://developerexperience.io/practices/pull-requests use value pull-requests

```suggestion
slug: api-first-development
```